bear-requirements.txt

```diff
@@ -36,6 +36,7 @@ rstcheck~=3.1
 safety~=1.8.2
 scspell3k~=2.0
 sqlparse~=0.2.4
+git+https://github.com/lipoja/URLExtract.git
```

Comment:
`URLExtract` has a pip package on https://pypi.org/project/urlextract/ . Could you explain why it's necessary to use the `git+https://` form?

Reply:
The version of URLExtract on pypi has not been updated in several months, and I need the latest version with the latest changes in order to implement this new feature.

Reply:
Sure thing! Just sent an email to the creator.

Reply:
The creator released a [new version on PyPi](https://pypi.org/project/urlextract/0.9/) yesterday, and I have updated `bear-requirements.txt` accordingly.